### PR TITLE
Add backend-agnostic video transformers example

### PR DESCRIPTION
**Pull request by soumik12345**
Converted the example [Video Classification with Transformers](https://keras.io/examples/vision/video_transformers/) to `keras_core`.

Linked Issue: https://github.com/keras-team/keras-core/issues/222

Colab Link: https://colab.research.google.com/drive/16NpSLw4RMfl5DTQdZRK2zBEvg6yD4F9o?usp=sharing

### CODE REVIEW

**Comment by soumik12345** [comment]
[Updated Colab](https://colab.research.google.com/drive/1h7_BQov7yPpBVFkxeEiRvsBPh-cbvJ1l?usp=sharing)
Working for both TF and JAX backends!

**Comment by soumik12345** [comment]
[Updated Colab](https://colab.research.google.com/drive/1h7_BQov7yPpBVFkxeEiRvsBPh-cbvJ1l?usp=sharing)
Working for TF, JAX, and Torch backends!

**Comment by soumik12345** [comment]
cc: @fchollet

File: examples/keras_io/vision/video_transformers.py [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+This example is a follow-up to the
+[Video Classification with a CNN-RNN Architecture](https://keras.io/examples/vision/video_classification/)
+example. This time, we will be using a Transformer-based model
+([Vaswani et al.](https://arxiv.org/abs/1706.03762)) to classify videos. You can follow
+[this book chapter](https://livebook.manning.com/book/deep-learning-with-python-second-edition/chapter-11)
+in case you need an introduction to Transformers (with code). After reading this
+example, you will know how to develop hybrid Transformer-based models for video
+classification that operate on CNN feature maps.
+
+This example requires TensorFlow 2.5 or higher, as well as TensorFlow Docs, which can be
```
Comment (fchollet) [inline examples/keras_io/vision/video_transformers.py:20]: Remove outdate message

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

**Review by fchollet — APPROVED** [review]
Thank you!

**Pull request merged**